royalties-registry/contracts/providers/RoyaltiesProviderArtBlocks.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.9 <0.8.0;
+pragma abicoder v2;
+
+import "@rarible/royalties/contracts/IRoyaltiesProvider.sol";
+import "./RoyaltyArtBlocks.sol";
+import "../lib/BpLibrary.sol";
+
+contract RoyaltiesProviderArtBlocks is IRoyaltiesProvider {
+    using SafeMathUpgradeable for uint;
+    using BpLibrary for uint;
+
+    uint96 artblocksPercentage = 250;
```

Comment:
if this isn't changing, then it should be constant to save gas. But I think it's good to be configurable